holoviews/tests/plotting/plotly/testcallbacks.py

```diff
@@ -96,6 +96,46 @@ def setUp(self):
                 'title': {'text': 'Figure Title'}}
         }).to_dict()
 
+        self.mapbox_fig_dict = go.Figure({
+            'data': [
+                {'type': 'scattermapbox', 'uid': 'first', 'subplot': 'mapbox'},
+                {'type': 'scattermapbox', 'uid': 'second', 'subplot': 'mapbox2'},
+                {'type': 'scattermapbox', 'uid': 'third', 'subplot': 'mapbox3'}
+            ],
+            'layout': {
+                'title': {'text': 'Figure Title'},
+            }
+        }).to_dict()
+
+        # Precompue pair so lat/lon, easting/northing, mapbox coord values
```

Comment:
```suggestion
        # Precompute pair so lat/lon, easting/northing, mapbox coord values
```

(Can't quite parse it, though!)